cpp/src/search/contains_column.cu

```diff
@@ -60,12 +61,16 @@ __device__ auto inline get_scalar_value(ScalarDView d_scalar)
 
 struct contains_scalar_dispatch {
   template <typename Type>
-  bool operator()(column_view const& haystack,
-                  scalar const& needle,
-                  rmm::cuda_stream_view stream) const
+  std::enable_if_t<!is_nested<Type>(), bool> operator()(column_view const& haystack,
```

Comment:
I believe that we talked about the same problem in this PR before :smile: 
Again, we can't use that style because we need to use device lambda in this function. This is a limitation of nvcc that doesn't allow device lambda in a function using `CUDF_ENABLE_IF` style.

Reply:
Please add a comment to note that. Apologies for mentioning this twice, just striving for consistency in the library.